### PR TITLE
[private-org-peribolos-sync] include whitelisted repos in peribolos sync

### PR DESCRIPTION
**Pull request by droslean**
/cc @petr-muller @openshift/openshift-team-developer-productivity-test-platform 

Signed-off-by: Nikolaos Moraitis <nmoraiti@redhat.com>

### CODE REVIEW

File: cmd/private-org-peribolos-sync/main.go [hunk 1]
```diff
@@ -159,8 +159,19 @@ func generateRepositories(gc gitHubClient, orgRepos map[string]sets.String, logg
 
 // getReposForPrivateOrg itterates through the release repository directory and creates a map of
 // repository sets by organization that promote official images.
-func getReposForPrivateOrg(releaseRepoPath string) (map[string]sets.String, error) {
+func getReposForPrivateOrg(releaseRepoPath string, whitelist map[string][]string) (map[string]sets.String, error) {
 	ret := make(map[string]sets.String)
+
+	for org, repos := range whitelist {
```
Comment (stevekuznetsov) [inline cmd/private-org-peribolos-sync/main.go:165]: You might want to make `map[string]sets.String` into an exported type and expose an `Insert(org, repo string)` function for it to DRY this out, so can share code.

Comment (droslean) [reply cmd/private-org-peribolos-sync/main.go:165]: Every tool is using its maps in a different manner, so there is no reason to expose the logic. Also, we don't need to insert anything in the existing whitelist.

**Pull request merged**